src/eventBus.js

```diff
@@ -238,9 +239,28 @@ const processUserRegistered = async (eventData) => {
       value: verificationObject.attribute_value,
       meta: userObject.profile || {},
     });
+
+    await fireEventToExternalEntity(eventType, data, neptuneData);
```

Comment:
@ShubhamJoshi-1994 This will run for everyone? can't we put check for `process.env.SEND_TO_SQS === "true"` here?

Reply:
@rajeshn95 this function has a check inside.

```
const fireEventToExternalEntity = async (
  eventType,
  data,
  neptuneData,
  externalEntity = null
) => {
  if (externalEntity && externalEntity.toLowerCase() === "sqs") {
    if (process.env.SEND_TO_SQS === "true") {
      await sendJob(data);
    }
  } else {
    if (process.env.SEND_EVENTS === "neptune") {
      await neptuneRepo.fireEvent(eventType, data, neptuneData);
    }
  }
};
```